src/System.Text.Json/tests/Utf8JsonReaderTests.TryGet.cs

```diff
@@ -238,9 +241,70 @@ public static void TestingNumbersInvalidConversionToInt32(string jsonString, dou
             {
                 if (json.TokenType == JsonTokenType.Number)
                 {
+                    #region TryGet calls
                     Assert.False(json.TryGetInt32Value(out int value));
                     Assert.True(json.TryGetDoubleValue(out double doubleValue));
                     Assert.Equal(expected, doubleValue);
+                    #endregion
+
+                    #region Get calls
+                    try
+                    {
+                        json.GetInt32();
+                        Assert.True(false, "Expected GetInt32 to throw FormatException.");
+                    }
+                    catch (FormatException) { /* Expected exception */ }
+                    catch (Exception except)
+                    {
+                        Assert.True(false, string.Format("Unexpected exception: {0}. Message: {1}.", except.GetType(), except.Message));
+                    }
+                    try
+                    {
+                        doubleValue = json.GetDouble();
+                    }
+                    catch (Exception except)
+                    {
+                        Assert.True(false, string.Format("Unexpected exception: {0}. Message: {1}.", except.GetType(), except.Message));
+                    }
+                    Assert.Equal(expected, doubleValue);
+                    #endregion
+                }
+            }
+
+            Assert.Equal(dataUtf8.Length, json.BytesConsumed);
+            Assert.Equal(json.BytesConsumed, json.CurrentState.BytesConsumed);
+        }
+
+        [Theory]
+        [InlineData("0.000")]
+        [InlineData("1e1")]
+        [InlineData("-2147483649")] // int.MinValue - 1
+        [InlineData("2147483648")]  // int.MaxValue + 1
+        public static void FormatException_Int32(string jsonString)
```

Comment:
Actually I didn't touch `TestingNumbersInvalidConversionToInt32` because it is also testing whether conversion to double is successful or not. I didn't understand why that is being done in method to test conversion to `Int32`. Likewise in `TestingNumbersInvalidConversionToInt64`. So should I remove the tests for conversion to double in those methods?

[Here](https://github.com/dotnet/corefx/blob/43759525e9a01fb9caedc0fb2281db5c65a16f65/src/System.Text.Json/tests/Utf8JsonReaderTests.TryGet.cs#L232)
```C#
        [InlineData("123456789012345678901", 123456789012345678901d)]
        public static void TestingNumbersInvalidConversionToInt32(string jsonString, double expected)
```
and [here](https://github.com/dotnet/corefx/blob/43759525e9a01fb9caedc0fb2281db5c65a16f65/src/System.Text.Json/tests/Utf8JsonReaderTests.TryGet.cs#L257)
```C#
        [InlineData("123456789012345678901", 123456789012345678901d)]
        public static void TestingNumbersInvalidConversionToInt64(string jsonString, double expected)
```

Reply:
Got it! Thanks!